core/src/main/kotlin/fr/sncf/osrd/api/pathfinding/constraints/LoadingGaugeConstraints.kt

```diff
@@ -3,38 +3,38 @@ package fr.sncf.osrd.api.pathfinding.constraints
 import fr.sncf.osrd.api.pathfinding.makePathProps
 import fr.sncf.osrd.graph.Pathfinding
 import fr.sncf.osrd.graph.PathfindingConstraint
+import fr.sncf.osrd.railjson.schema.rollingstock.RJSLoadingGaugeType
```

Comment:
I'd rather avoid having RJS types leak into the implementation

Reply:
I tried to move from the RJS type to the non-RJS type when parsing a rolling stock, but that didn't quite work out. The non-RJS type is a `StaticIdx` and all of the rolling stock related code is in java, we'd lose a lot of readability

Nothing really changed in this file, the type is now explicit but it was still used before

Reply:
fine